src/pontibus/protocols/solvation/base.py

```diff
@@ -242,18 +264,22 @@ def _get_alchemical_system(
         alchemical_indices : list[int]
           A list of atom indices for the alchemically modified
           species in the system.
+        alchemical_settings : AlchemicalSettings
+          Settings controlling how the alchemical system is built.
```

Comment:
(blocking) I'm pretty sure this should be in the `Parameters` section 10 lines above, not the `Returns`